hunts/src/App.js

```diff
@@ -1,10 +1,26 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { createRoot } from "react-dom/client";
 import { Provider } from "react-redux";
 import store from "./store";
 import { HuntViewMain } from "./HuntViewMain";
 
+const sendTokenToChromeExtension = ({ extensionId, huntId }) => {
+  chrome.runtime.sendMessage(extensionId, { huntId }, (response) => {
```

Comment:
think this should be wrapped in `if (chrome && chrome.runtime) {`

More generally ideally the extension could handle this kind of thing itself in a content script I think, but not a huge deal.